jax/numpy/lax_numpy.py

```diff
@@ -498,6 +498,18 @@ def op(*args):
 logical_xor = _logical_op(onp.logical_xor, lax.bitwise_xor)
 
 
+@_wraps(onp.rint)
+def rint(x):
+  if issubdtype(_dtype(x), complexfloating):
+    return lax.complex(rint(lax.real(x)), rint(lax.imag(x)))
+  y = floor(x)
```

Comment:
Doesn't `lax.round` have the semantics you want already for the float case? (I'm not 100% sure). Can we simplify this?

Reply:
I just checked and it doesn't (weirdly). It rounds using the usual rule of rounding upwards in the case of a tie (i.e. not rounding to the nearest even).

Reply:
Fair enough. Is the documentation on `lax.round` clear on this point? Maybe we should add an example?

Also, there's a `_round_to_nearest_even` in this file. What about that one? :-)

Reply:
Can still get a slight simplification by using lax.round, see upcoming commit.

Reply:
Actually turns out I can't figure out a way to simplify using lax.round 😐.

Reply:
What about `_round_to_nearest_even` vs this implementation?

Reply:
will use _round_to_nearest_even, thanks for pointing that out!

Reply:
That may also fix the travis failure. When using the operator syntax you need to be careful about type promotion.

Reply:
Yeah, the travis failure was due to the behavior on boolean types not matching Numpy, but only on py36. I've just disabled testing on boolean types.

Reply:
Gonna resolve this since I'm now using _round_to_nearest_even.